docs/trace_collection.md

```diff
@@ -279,55 +520,125 @@ To provide a continuous trace inside a RxJava stream you need to follow the step
    ```
 2. Then in your project open a scope when the Observable is subscribed and close it when it completes. Any span
    created inside the stream operators will be displayed inside this scope (parent Span):
-   ```kotlin
-   Single.fromSupplier{}
+
+{{< tabs >}}
+{{% tab "Kotlin" %}}
+```kotlin
+Single.fromSupplier{} 
         .subscribeOn(Schedulers.io())
-        .map { 
+        .map (
```

Comment:
it was a correct usage of `{` before, Kotlin is `map { item -> ... }`, Java is `map(item -> { ....} )`, same for the calls below

Reply:
yes...this was nit changed is just that for `map` in kotlin you don't have to specify it as you can use `it`

Reply:
I mean the outer brackets are wrong for `map`, `doOnSubscribe`, `donFinally` are wrong